i18n/about/staff.i18n.js

```diff
@@ -0,0 +1,242 @@
+import { genI18nMessages } from '~/utils/i18n.utils'
+
+export default genI18nMessages({
+    'en-us': {
+        title: 'Staff',
+        contents: [
+            {
+                organization: 'Chairperson',
+                members: ['Wei Lee'],
+            },
+            {
+                organization: 'Program',
+                members: [
+                    'Winnie',
+                    'Pochun',
+                    'Petertc',
+                    'JunWei Song',
+                    'Annie Bu',
+                    'Jordan Su',
+                    'Gina',
+                    'Kyle',
+                    'Yider',
+                    'Tony',
+                    '黃坤賢',
+                    'TingYi',
+                    'Winfred',
+                    'Shihchi',
+                    'Huei Lin',
+                    'Benson',
+                    'Jason',
+                    'Kir',
+                    'Leonard',
+                    'Lance',
+                    'Shirley',
+                ],
+            },
+            {
+                organization: 'Development',
+                members: [
+                    'Matt Wang',
+                    'Kaka Lin',
+                    'Ethan Lin',
+                    'Max',
+                    'Set Mao',
+                    'Maliao',
+                    'Flynn',
+                    'Alice Chang',
+                    '泰瑋',
+                    'Josix',
+                    'Hane',
+                    'SHY',
+                    'Phil',
+                    'Benson',
+                ],
```

Comment:
update for dev team
```suggestion
                members: [
                    'Matt Wang',
                    'Kaka Lin',
                    'Ethan Lin',
                    'Max',
                    'Set Mao',
                    'Maliao',
                    'Flynn',
                    'Alice Chang',
                    '泰瑋',
                    'Josix',
                    'Hane',
                    'SHY',
                    'Phil',
                    'Mozix',
                    'cloudy',
                    'GaryPai',
                    'Jacky',
                    'Benson',
                ],
```

Reply:
Also, consider divide the section for dev team into three since we have 3 squad under dev team now.

Reply:
Thank you. I have changed the section for dev team into three  (f1f92d9).